requirements.txt

```diff
@@ -0,0 +1,18 @@
+certifi==2019.6.16
```

Comment:
Do we need all these or that guy’s requirement.txt is enough?

Reply:
These are all pip freeze , they are auto generated.its same as the other guys .